testutil/duration.go

```diff
@@ -1,5 +1,3 @@
-//go:build !windows
```

Comment:
This change has me worried due to previously not only Windows runners being underpowered but disk I/O was always slower on Windows too. Feel free to have a go at this but I'd advice to be prepared to revert it 😄